docs/blocks/authentication/ForgotPassword.story.tsx

```diff
@@ -355,7 +355,7 @@ export const ResetPassword = () => {
           </a>
         </Stack>
         <div className="w-full flex flex-col grow items-center justify-center text-center pt-2 mb-14">
-          <div className="p-3 border border-panel-secondary-content rounded mb-3">
+          <div className="p-3 border dark:border-waterloo light:border-gray-700 rounded mb-3">
```

Comment:
This couples the implementation to the specific color. Do we not have a abstraction for this?

Reply:
yea - i can probably use one of the accent colors. i'll update